packages/next/build/webpack/plugins/flight-types-plugin.ts

```diff
@@ -27,12 +27,12 @@ check<IEntry, TEntry>(entry)
 
 type PageParams = Record<string, string>
 interface PageProps {
-  params?: PageParams
-  searchParams?: Record<string, string | string[]>
+  params: any
+  searchParams: any
```

Comment:
those props should still be optional?
```suggestion
  searchParams?: any
```

Reply:
No these shouldn't...